### DISCUSSION TITLE
Unable to add to 'Skip'

### DISCUSSION THREAD

**Discussion by merc07** [egzumer/uvk5-chirp-driver#30] [Q&A]
Are we able to add to 'Skip' channels when scanning?

![image](https://github.com/egzumer/uvk5-chirp-driver/assets/33531315/ff78eef0-b84a-4cb3-8888-43c09e8af834)

**Comment by f4axk** [comment]
To “Skip” when scanning
scanlist none

![none](https://github.com/egzumer/uvk5-chirp-driver/assets/68162981/6b13e249-bbfa-4bc1-8bf1-439b1bf7ee9e)

**Comment by merc07** [comment]
You can set the radio to scan List 1, List 2 or * which scans all channels
Even if set to none, the radio will scan that channel in the * scan mode
I would have thought the 'skip' column was there for that purpose ?

**Reply by f4axk** [reply]
if you have a uv5r skip radio it is possible to activate skip.
if radio uvk5 none replaces skip.

possibility to exclude a channel with key 5 noaa


5 NOAA
in channel mode - toggles scan lists that the selected channel is assigned to. You will see I and II symbols changing on the right side of the channel label

https://github.com/egzumer/uv-k5-firmware-custom/wiki/Button-functions

**Comment by egzumer** [comment]
"Skip" is a generic CHIRP function, which is not supported by the driver. At the first glance if used would be limiting to the scan settings that  quansheng offers (there are 2 scan lists, not one). Use Scanlists option as suggested.
